src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -28,8 +28,15 @@ config:
 tests:
     - label: "Precondition: Commission DUT to TH1"
       verification: |
-          Once DUT reach the  commissionable state send the following command on TH1:
-          pairing onnetwork 1 20202021
+          - DUT should be commissioned onto TH1, TH2, and TH3.
+          - The Scene Table capacity for a given fabric SHALL be less than half (rounded down towards 0) of the Scene Table entries (as indicated in the SceneTableSize attribute).
+          - MaxRemainingCapacity is (SceneTableSize-1)/2.
+          - TH1, TH2, and TH3 should be on separate, distinct fabrics.
+
+          Send the below command in respective DUT
+          ./chip-all-clusters-app
+
+          Once DUT reach the  commissionable state send the following command  ./chip-tool pairing onnetwork 1 20202021 command on TH1.
```

Comment:
In precondition step, row numbers 48, 62, 74, 88. `./chip-tool` prefix has to be added for the chip-tool Command. It should be `./chip-tool  pairing open-commissioning-window 1 1 400 2000 3841` instead of  `pairing open-commissioning-window 1 1 400 2000 3841`.

Reply:
Added in last commit.